platform/darwin/src/MGLStyle.h

```diff
@@ -263,6 +263,13 @@ MGL_EXPORT
  */
 @property (nonatomic, assign) BOOL performsPlacementTransitions;
 
+/**
+A set containing user-specified source identifiers for point features available for accessibility. The point features must have a `name` attribute that matches those specified by <a href="https://www.mapbox.com/vector-tiles/mapbox-streets-v8/#overview">Mapbox Streets</a> source and belong to a `MGLVectorStyleLayer`.
+ 
+This set does not include Mapbox Streets source identifiers included by default.
+*/
+@property (nonatomic) NSSet *accessiblePlaceSourceIdentifiers;
```

Comment:
Rename the property to `accessiblePlaceSourceLayerIdentifiers` to reflect the fact that it contains source _layer_ identifiers, not source identifiers.

It’s also worth pointing out that this property only applies to MGLVectorTileSource objects or vector tile sources in style JSON. It isn’t possible for an MGLShapeSource, MGLComputedShapeSource, or GeoJSON source in style JSON to have source layers. If we want shape sources to participate in this feature, we’d have to add a second property that is a set of MGLShapeSource and MGLComputedShapeSource objects to also include.